### PR TITLE
Automatic relay of unrelayed packets

### PR DESCRIPTION
**Pull request by teet71**

### CODE REVIEW

File: client/configuration.ini.sample [hunk 1]
```diff
@@ -35,6 +35,8 @@ connection-retry-time=5
 # The callsign is made up of upper-case alpha and numeric ASCII characters only, maximum length of 6.
 satellite-src = CHANGE_ME
 
+
```
Comment (nowittyusername) [inline client/configuration.ini.sample]: Why do we need this extra whitespace here?

**Review by nowittyusername — CHANGES_REQUESTED** [review]

**Comment by nowittyusername** [comment]
Please address the comment I made. Secondly, isn't there a better way to check for True or False than the following?

str(self.config.get_conf("Mission Control", "relay-enabled")) != "True"

**Comment by nowittyusername** [comment]
Finally, there seems to be something up with some threads not being properly terminated.

```
WARNING:tnc_pool:Could not initialize a TCP connection to localhost:8001
INFO:tnc_pool:Retrying TNC Main connection in 5 seconds...
2020-11-19 10:59:06,170 kiss INFO kiss.classes.start:248 - Connected to ('localhost', 8001)
^C2020-11-19 10:59:36,200 kiss INFO kiss.classes.stop:238 - KISS TCP interface closed.
^CException ignored in: <module 'threading' from '/usr/lib/python3.8/threading.py'>
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 1388, in _shutdown
    lock.acquire()
KeyboardInterrupt: 
```

**Comment by nowittyusername** [comment]
In one terminal window I have a TNC waiting for connections
```
Dire Wolf version 1.5

Reading config file /home/user/direwolf.conf
Audio input device for receive: udp:7355  (channel 0)
Audio out device for transmit: default  (channel 0)
Channel 0: 9600 baud, K9NG/G3RUH, +, 48000 sample rate x 2.
The ratio of audio samples per sec (48000) to data rate in baud (9600) is 5.0
Increasing the sample rate should improve decoder performance.
Note: PTT not configured for channel 0. (Ignore this if using VOX.)
Ready to accept AGW client application 0 on port 8000 ...
Ready to accept KISS TCP client application 0 on port 8001 ...
```

In a second one I start the client with `python3 src/main.py` however it never connects with the TNC (the correct port is defined in the .ini file) but the program hangs here 
```
INFO:__main__:Using configuration from: /home/user/telemetry_forwarder/client/src/../configuration.ini
INFO:db_interface:Initializing database at /home/user/telemetry_forwarder/client/src/../telemetry.db
```

After doing a ctrl+c I get the following:
```
^C^CWARNING:file_logger:Connection to tle endpoint failed
Traceback (most recent call last):
  File "src/main.py", line 118, in <module>
    main(sys.argv[1:])
  File "src/main.py", line 76, in main
    file_logger = FileLogger(conf, '../logs/', "log")
  File "/home/user/telemetry_forwarder/client/src/file_logger.py", line 22, in __init__
    self.file_logger = logging.FileHandler(logfile)
  File "/usr/lib/python3.8/logging/__init__.py", line 1143, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.8/logging/__init__.py", line 1172, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/user/telemetry_forwarder/logs/log_packets.log'
```
@teet71, please comment on this how to fix

**Comment by nowittyusername** [comment]
So I am guessing it's blocking on the tle endpoint connection. I would suggest you to make a timeout for it and disable whatever the tle controls until a restart. That would most likely be the easiest solution.

**Comment by teet71** [comment]
@Risto-R

**Comment by nowittyusername** [comment]
Tests fail

AssertionError: 5000 != '5000'

**Review by nowittyusername — APPROVED** [review]

**Pull request merged**